index.bs

```diff
@@ -1385,38 +1383,45 @@ Build a composed graph up to a given output operand into a computational graph a
   <summary>
     The <dfn method for=MLGraphBuilder>build(|outputs|)</dfn> method steps are:
   </summary>
-    1. Let |promise| be [=a new promise=].
+    1. If |outputs| is empty, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. [=map/For each=] |name| → |operand| of |outputs|:
+        1. If |name| is empty, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+        1. If [=MLOperand/validating MLOperand=] given |operand| and [=this=] returns false, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+        1. If |operand| is in [=this=]'s [=MLGraphBuilder/graph=]'s [=computational graph/inputs=] or [=computational graph/constants=], then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. Let |operands| be a new empty [=/set=].
+    1. Let |queue| be a new [=queue=] containing |outputs|'s [=map/values=].
+    1. While |queue| is not [=queue/empty=]:
+        1. [=queue/Dequeue=] |operand| from |queue|.
+        1. [=set/Append=] |operand| to |operands|.
+        1. [=list/For each=] |input| of |operand|.{{MLOperand/[[operator]]}}'s [=operator/inputs=]:
+            1. [=queue/Enqueue=] |input| to |queue|.
+    1. If any {{MLOperand}}s in |operands| have the same non-empty {{MLOperand/[[name]]}}, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
```

Comment:
This step may just check operand in `inputs`.

Reply:
Done in ba9b7bb2ebe684ef675766e2a3982aecbf0f79c9